### PR TITLE
class Flags  to use as enum

### PR DESCRIPTION
**Pull request by Alexqwesa**
class Flags  to use as enum

### CODE REVIEW

**Comment by cedvdb** [comment]
How were those strings generated is there a script to rebuild it when new files are added ?

**Comment by Alexqwesa** [comment]
> How were those strings generated is there a script to rebuild it when new files are added ?

yes, I have used `build` package to generate it, I will make new pull request later...  

Please regard this as a proposal for your consideration...

File: lib/src/flags.dart [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+  static const String CX = 'cx';
+  static const String CY = 'cy';
+  static const String CZ = 'cz';
+  static const String DE = 'de';
+  static const String DJ = 'dj';
+  static const String DK = 'dk';
+  static const String DM = 'dm';
+  static const String DO = 'do';
+  static const String DZ = 'dz';
+  static const String EASTERISLAND = 'easter_island';
```
Comment (cedvdb) [inline lib/src/flags.dart:75]: I'd use the same separator here

**Comment by cedvdb** [comment]
> Please regard this as a proposal for your consideration...
 
 I'm not sure I understand what you meant by this :) But yeah I just wonder if it's going to be used and therefor worth adding. Maybe ? 

I figured most people would use this dynamically but maybe that's just my use case.

**Pull request closed without merging**